docs/documentation/server_admin/topics/sessions/timeouts.adoc

```diff
@@ -24,18 +24,18 @@ image:images/sessions-tab.png[Sessions Tab]
 |This setting is similar to the standard SSO Session Max but specific to *Remember Me* logins. Users can specify longer sessions when they click *Remember Me* when logging in. This setting is an optional configuration and, if its value is not greater than zero, it uses the same session lifespan as the SSO Session Max configuration.
 
 |Client Session Idle
-|If the user is inactive for longer than this timeout, refresh token requests bump the idle timeout. This setting specifies a shorter idle timeout of refresh tokens than the session idle timeout, but users can override it for individual clients. This setting is an optional configuration and, when set to zero, uses the same idle timeout in the SSO Session Idle configuration.
+|Idle timeout for the client session. If the user is inactive for longer than this timeout, the client session is invalidated and the refresh token requests bump the idle timeout. This setting never affects the general SSO user session, which is unique. A SSO user session has zero or more client sessions, apps the user has requested access/tokens to. This value should specify a shorter idle timeout than the *SSO Session Idle*. Users can override it for individual clients in the *Advanced Settings* client tab. This setting is an optional configuration and, when set to zero, uses the same idle timeout in the SSO Session Idle configuration.
```

Comment:
I am not certain about the meaning of this sentence:  An SSO user session has zero or more client sessions, apps the user has requested access/tokens to. 
I suggest clarifying it.

Reply:
I'm just trying to explain that the user session is parent of several client sessions. One client session is created for every different client app the user logs in. This `Client Session Idle` option is only for the child/client session, not for the parent user session. So the client session will disappear sooner than the user session if we use a shorter value here. And that's the reason it should be  shorter, because a larger value in this option is meaningless, when the parent user session disappears, all the client sessions attached to it are also inaccessible.

Reply:
I modified the phrase trying to make it clearer.